cli.js

```diff
@@ -27,19 +36,34 @@ const cli = meow(`
 
 	Options
 	  --copy -c  Copy the first emoji to the clipboard
+	  --skin-tone -s  Set the default skin tone of the emoji (0 to 5)
 
 	Run it without arguments to enter the live search
+	Use Up/Down keys during live search to change the skin tones
 `, {
 	boolean: [
 		'copy'
 	],
 	alias: {
-		c: 'copy'
+		c: 'copy',
+		s: 'skinTone'
 	}
 });
 
+if (typeof cli.flags.skinTone === 'number') {
```

Comment:
@satazor I agree, but I think it would be better to set the default in the minimist options. https://github.com/substack/minimist